google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/sdk/CloudSdkPanel.java

```diff
@@ -200,8 +200,14 @@ public String getCloudSdkDirectoryText() {
     return cloudSdkDirectoryField.getText();
   }
 
+  /**
+   * Sets the Cloud SDK directory text. Performs no action if the current value of the Cloud SDK
+   * directory field is already equal to the path.
+   */
   public void setCloudSdkDirectoryText(String path) {
-    cloudSdkDirectoryField.setText(path);
+    if (!cloudSdkDirectoryField.getText().equals(path)) {
```

Comment:
I don't think so, but it can't hurt to be safe here since this is a third-party API. Good catch.